networking/src/NTP/Client.hs

```diff
@@ -147,16 +154,20 @@ startSend :: NtpMonad m => [SockAddr] -> NtpClient m -> m ()
 startSend addrs cli = do
     let timeout = ntpResponseTimeout (ncSettings cli)
     let poll    = ntpPollDelay (ncSettings cli)
-    logDebug "Sending requests"
-    liftIO . atomically . modifyTVarS (ncState cli) $ identity .= Just []
-    let sendRequests = forConcurrently addrs (flip doSend cli)
-    withAsync sendRequests $ \_ ->
-        () <$ threadDelay timeout
-
-    logDebug "Collecting responses"
-    handleCollectedResponses cli
-    liftIO . atomically . modifyTVarS (ncState cli) $ identity .= Nothing
-    liftIO $ threadDelay (poll - timeout)
+
+    _ <- concurrently (threadDelay poll) $ do
```

Comment:
Aren't we simply moving the problem one layer up, with this call to `concurrently`? This will still wait `poll` time before returning, which is capping the endpoint to take 5 secs, which I think is too long for our current use case.